Add redenomination page (#740)
w3f/polkadot-wiki: docs/redenomination.md

---
id: redenomination
title: Redenomination of DOT
sidebar_label: Redenomination of DOT
---

> **The DOT redenomination is taking place on 21st August, now to be known as Denomination Day, at

Reviewer: It might be easier for new users to understand renomination if there was a definition and a reason why it took place.

Reviewer: Something like this would make it easier for people that are trying to integrate in this industry to understand :)

Author: I rewrote the first couple sentences, hopefully making an easier on-ramp for new readers.

block #1,248,328. This block will happen in Europe's early afternoon, China's evening and America's
morning.**

> The DOT was voted by the community to be redenomination from its initial representaiton of 10e12

Reviewer 2: ```suggestion
The [DOT](learn-dot) was voted by the community to be redenominated from its initial representation of 10e12
```

Plancks to 10e10 Plancks. In other words, the DOT went from having twelve decimal places of
precision to only ten decimal places of precision. The overall effect of this change was that the

> amount of Polkadot's smallest unit, the Planck remained constant, while DOT balances for all holders

Reviewer 2: ```suggestion
amount of Polkadot's smallest unit, the Planck, remained constant, while DOT balance for all holders
```

> was increased by a factor of ten.

Reviewer 2: ```suggestion
was increased by a factor of one hundred.
```

```
   Before the change the decimal was here
   v
  1.000000000000 DOT

  100.0000000000 DOT
     ^
     After the change the decimal is here
```

## Origins

The initial vote for redenomination occurred as a [referendum][referendum 52] on the Kusama

> blockchain. The referndum was summarized as having four effects if approved by KSM holders.

Reviewer 2: ```suggestion
blockchain. The referendum was summarized as having four effects if approved by KSM holders.
```

>   network as before the change.
> - The precision of DOT will change from 12 decimal places to 10 decimal places.
> - The main benefit of this change is to avoid using small decimals when dealing with DOT and to
>   achieve an easier calculation system.

The initial referendum was proposed prior to the Polkadot genesis block under the assumption that
making a redenomination would be simpler before the Polkadot chain was live. However, many in the
community pointed out the disconnect of the two networks and how it was unfair for holders of DOT to
be impacted by a vote by a different token holder set. For this reason, the Web3 Foundation [decided
to make a new vote on Polkadot][blog 1] when it went live, although the Kusama vote ended with a
majority in favor of the redenomination change.

A bolded quote from the Foundation read:

> However, given the non-negligible amount of opposition, including from some within the ranks of
> Web3 Foundation and Parity, **the Foundation decided that we cannot, in good faith, sponsor
> redenomination at present.**

## The Vote

After the genesis block of Polkadot was created and the network was running with a decentralized

> community of validators securing the network, it was decided to put the redenomination topic up to a

Reviewer: Who was it decided by? It might be helpful to include this for new users

> vote again. This time, the vote was explicitly binding - whereas the vote on Kusama was of course

Reviewer: Perhaps including a link here to a definition of binding will be useful for new learners

non-binding being at best a way to capture of a signal and not able to affect the Polkadot chain
directly.

Due to the feedback received during the Kusama referendum, the Polkadot vote was to be held in an
approval vote style, with four available options. DOT holders could issue votes for any
configuration of the four options: no change, a change of 10x, a change of 100x, or a change of
1000x. The vote logic was contained in a specially-built Substrate pallet that was included in
Polkadot's runtime for this specific purpose.

> The important notes of the Polkadot vote are as follows:

Reviewer: ```suggestion
The important notes of a Polkadot vote are as follows:
```

Author: In this case, I'm referring to a specific vote on Polkadot so it should be "the Polkadot vote".

> - No discretionary lock-voting is in place; all DOT used to vote counts the same.
> - Voting is made on a per-account basis; a single account must all vote the same way and cannot
>   split its vote.
> - This vote does not affect any economics of the Polkadot platform. Staking rewards, inflation,
>   effective market capitalization and the underlying balances of every account remain completely
>   unchanged. It is “merely” about what units we use to denominate the balances into “DOT” for the
>   purpose of display.

With a voting period of two weeks set- the redenomination was now in the hands of the Polkadot
community for a final, binding decision.

## The Outcome

![redenomination](assets/redenomination.png)

After two weeks of voting, the results of the redenomination vote were tallied. With around
one-third of the total DOTs of the network having cast a vote, the redenominaton proposal passed
with 86% of the voters favoring a x100 factor increase (or two decimal places of precision loss).

Polkadot's redenomination is taking place on 21st August, now to be known as Denomination Day, at
block #1,248,328

### What This Means for the Community

> If you're a DOT holder, or user of the network than you do not need to take any action. The DOT

Reviewer 2: ```suggestion
If you're a DOT holder or user of the network, then you do not need to take any action. The DOT
```

redenomination is a purely front-end change. You will still hold the same amount of Plancks after
the change, but now it will appear that you hold 100x more DOTs. This change applies proportionally
to everyone.

### What This Means for Builders of Tools

> If you are a builder of tool that consumes the `@polkadot/api` package - then there should be no

Reviewer 2: ```suggestion
If you are the builder of  a tool that consumes the `@polkadot/api` package - then there should be no
```
